interface/main/calendar/find_appt_popup.php

```diff
@@ -353,10 +337,10 @@ function setappt(year,mon,mday,hours,minutes) {
 <body class="body_top">
 <div class="container-responsive">
 <div id="searchCriteria">
-<form class="form-inline" method='post' name='theform' action='find_appt_popup.php?providerid=<?php echo attr($providerid) ?>&catid=<?php echo attr($input_catid) ?>'>
+<form class="form-inline" method='post' name='theform' action='find_appt_popup.php?providerid=<?php echo attr_url($providerid) ?>&catid=<?php echo attr_url($input_catid) ?>'>
```

Comment:
hi @attritionorg , This function is also incorporating urlencode() to the standard attr() escaping function (xss)